### PR TITLE
Load-balancer how to

### PR DESCRIPTION
**Pull request by evilnick**
I originally wrote a much larger doc but I have stripped all that content out and will save for an explanation page (we should also cover using external load-balancers first before adding that.
So, this is fairly straightforward.

### CODE REVIEW

File: docs/src/howto/networking/default-loadbalancer.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# How to use the default load-balancer
+
+Canonical Kubernetes includes a default load-balancer. As this is not an
+essential service for all deployments, it is not enabled by default. This guide
+explains how to configure and enable the loadbalancer.
```
Comment (eaudetcobello) [inline docs/src/howto/networking/default-loadbalancer.md:5]: load-balancer or loadbalancer?

File: docs/src/howto/networking/default-loadbalancer.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+sudo k8s get load-balancer
+```
+This should output a list of values like this:
+
+
+- `cidrs` - a list containing [cidr] definitions of the pool of IP addresses to use
+- `l2-mode` - whether L2 mode (failover) is turned on
+- `l2-interfaces`  - optional list of interfaces to announce services over( defaults to all)
+- `bgp-mode` - whether BGP mode is active.
+- `bgp-local-asn` - the local Autonomous System Number(ASN)
```
Comment (eaudetcobello) [inline docs/src/howto/networking/default-loadbalancer.md:38]:
```suggestion
- `bgp-local-asn` - the local Autonomous System Number (ASN)
```

**Review by eaudetcobello — COMMENTED** [review]
Cool! Just a couple typos

**Review by kwmonroe — APPROVED** [review]
couple drive-by tweaks.  otherwise lgtm.

**Pull request merged**